src/Umbraco.Web.UI.Client/src/packages/block/block-list/workspace/views/block-list-type-workspace-view.element.ts

```diff
@@ -75,6 +75,7 @@ export class UmbBlockListTypeWorkspaceViewSettingsElement extends UmbLitElement
 				<umb-property
 					label=${this.localize.term('blockEditor_forceHideContentEditor')}
 					alias="forceHideContentEditorInOverlay"
+					description="Hide the content edit button and the content editor from the Block Editor overlay."
```

Comment:
Looks a good addition, but it would be nice if the description was localized, in the same way that the `label` attribute is a couple of lines above.  Do you think you could set it up that way?  You'll need to create translation keys in, at least, the `en.ts`, `en-us.ts` and `da-da.ts` files.